examples/gallery/histograms/histogram.py

```diff
@@ -19,19 +20,39 @@
 
 fig = pygmt.Figure()
 
-fig.histogram(
-    data=data,
-    # Define the frame, add a title, and set the background color to
-    # "lightgray". Add labels to the x-axis and y-axis
-    frame=["WSne+tHistogram+glightgray", "x+lElevation (m)", "y+lCounts"],
-    # Generate evenly spaced bins by increments of 5
-    series=5,
-    # Use "red3" as color fill for the bars
-    fill="red3",
-    # Use the pen parameter to draw the outlines with a width of 1 point
-    pen="1p",
-    # Choose histogram type 0, i.e., counts [Default]
-    histtype=0,
-)
+with fig.subplot(
+    nrows=1,
+    ncols=2,
+    figsize=("13.5c", "5c"),
+    title="Histograms",
+):
+    with fig.set_panel(panel=0):
+        fig.histogram(
+            data=data,
+            # Define the frame, add title and set background color to
+            # "lightgray", add labels for x and y axes
+            frame=["WSne+glightgray", "x+lElevation in m", "y+lCounts"],
```

Comment:
The main title is now set via the `title` parameter of `pygmt.Figure.subplot`.

```suggestion
            # Define the frame and set background color to "lightgray",
            # add labels for x and y axes
            frame=["WSne+glightgray", "x+lElevation in m", "y+lCounts"],
```